### DISCUSSION TITLE
Isomorphic Clerk Question

### DISCUSSION THREAD

**Discussion by targetlucked69** [clerk/javascript#3386] [Support request - DO NOT OPEN HERE]
Hi! First of all, thank you for this really awesome library.

I wanted to learn more about it so I took a deep dive on the source code, and I have one question:

What is [IsomorphicClerk](https://github.com/clerk/javascript/blob/main/packages/react/src/isomorphicClerk.ts) for? Why did you wrap the `Clerk` instance with another one? Is it so that you can instantiate it in SSR without getting errors?

Thank you!

**Comment by jescalan** [comment]
https://github.com/orgs/clerk/discussions/3829
